.github/ISSUE_TEMPLATE/Update A-enviar-software-para-revision.md

```diff
@@ -0,0 +1,105 @@
+---
+name: Enviar software para revisión
+about: Usa esta plantilla para enviar un software a revisar en Español (Experimental)
+---
+
+
+Autor Encargado: nombre (@github_handle)  
+Repositorio:   
+Versión Enviada:   
+Editor: TBD  
+Revisor 1: TBD  
+Revisor 2: TBD  
+Archivo: TBD  
+Versión Aceptada: TBD   
+
+---
+
+
+
+-   Pega el archivo DESCRIPTION completo dentro del siguiente bloque de código.
+
+```
+
+```
+
+
+## Alcance 
+
+- Por favor, indica qué categoria(s) aplican a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés=](https://ropensci.github.io/dev_guide/policies.html#package-categories). Por favor, tilda todas las apropiadas. Si no estás seguro, te sugerimos que comiences un pre-envío.
```

Comment:
I like the use of "tu" not "Usted". Was it a conscious decision?

Reply:
I tried to write in neutral Spanish, which is a variation of the latinamerican Spanish.